index.html

```diff
@@ -50,5 +50,174 @@ <h2>Let's connect</h2>
             </ul>
         </section>
     </div>
+    <div class="works">
+        <section class="portfolio">
+            <div>
+                <img src="Assets/Tonic1.png" alt="Portfolio image">
+            </div>
+            <div>
+                <h1>Tonic</h1>
+                <div class="summary">
```

Comment:
Your design
![image](https://github.com/firepicaso/RWD_Portfolio/assets/78251927/a5c79a9d-1659-4ead-b898-8978a9711771)

Figma
![image](https://github.com/firepicaso/RWD_Portfolio/assets/78251927/a3729f8c-60e0-4b50-b65f-3839183206fa)

- I would suggest that we use a header tag `hx` (h2, h3, ...) for the title (e.g., "Tonic"). The `<h1>` to `<h6>` tags are used to define HTML headings. However, `<h1>` defines the most important heading (i.e.  The main title of the page). `<h6>` defines the least important heading.  [Check this link](https://www.reliablesoft.net/h1-tag/) for more info

- Kindly use the same color as the design on Figma for the header so as to stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 💪